macros/utils/table_operations/create_or_replace.sql

```diff
@@ -25,3 +25,12 @@
     {% do run_query(dbt.create_table_as(temporary, relation, sql_query)) %}
     {% do adapter.commit() %}
 {% endmacro %}
+
+{% macro athena__create_or_replace(temporary, relation, sql_query) %}
+    {% set drop_query %}
```

Comment:
TODO: `relation` doesn't have `type` for some reason when this is called during the initial run to build elementary tables. `drop_relation` fails as it checks for type.